packages/idealo.design/src/server/index.js

```diff
@@ -199,6 +201,18 @@ app.put('/api/blogposts', isAuthenticated, async (req, res) => {
   return res.json(createdBlogpost);
 });
 
+app.delete('/api/blogposts/delete', isAuthenticated, async (req,res) => {
```

Comment:
Sorry for the nitpicking but I'd like to keep the API as RESTful as possible. In this case, it means to make the endpoint look like:
`DELETE /api/blogposts/:blogpost_id`. However, I like that you're already using http DELETE.